docs/templates.md

```diff
@@ -93,6 +93,7 @@ This type cannot be printed. The following methods are defined.
 * `parents() -> List<Commit>`
 * `author() -> Signature`
 * `committer() -> Signature`
+* <a name="commit.signature"></a>`signature() -> Option<CryptographicSignature>`
```

Comment:
nit: not sure if it's common to add anchor in markdown. Maybe we can link to the `CryptographicSignature type` section instead?

Reply:
I don't understand.
We are linking from the [Commit Signature Verification section in config.md](https://github.com/pylbrecht/jj/blob/e3aebadb8a86d5d9a6e44eb1d359f2dafe588390/docs/config.md#L1030) to here. That paragraph in `config.md` also links to the `CryptographicSignature` type.
That way you effortlessly get from the [Commit Signature Verification section](https://github.com/pylbrecht/jj/blob/e3aebadb8a86d5d9a6e44eb1d359f2dafe588390/docs/config.md#L1030) to the template functions.

Or are you suggesting to remove the `commit.signature` anchor entirely and only link to the `CryptographicSignature` type?

Reply:
If you leave the anchor, you should not be using a deprecated attribute (`name`). The `id` attribute is the standard, and the id should be unique on a page.